stdlib/multiprocessing/managers.pyi

```diff
@@ -66,6 +65,62 @@ class ValueProxy(BaseProxy, Generic[_T]):
     if sys.version_info >= (3, 9):
         def __class_getitem__(cls, item: Any) -> GenericAlias: ...
 
+class DictProxy(BaseProxy, MutableMapping[_KT, _VT]):
```

Comment:
Also `__contains__`, `clear`, `popitem`, `setdefault`, `update`.

(Or are those inherited? In that case, why do we have `__len__`?)

Reply:
All inherited from `MutableMapping`. `MutableMapping.__len__` is an abstract method, so must be overridden here in order to make this class concrete.